project/Dependencies.scala

```diff
@@ -21,7 +21,8 @@ object Dependencies {
   def metacp = "org.scalameta" %% "metacp" % scalametaV
   def semanticdbPluginLibrary = "org.scalameta" % "semanticdb-scalac-core" % scalametaV cross CrossVersion.full
   def scalameta = "org.scalameta" %% "scalameta" % scalametaV
-  def scalatest = "org.scalatest" %% "scalatest" % "3.0.8"
+  def bijectionCore = "com.twitter" %% "bijection-core" % "0.9.7"
+  def scalatest = "org.scalatest" %% "scalatest" % "3.1.2"
```

Comment:
Is there a reason this upgrade is needed? The upgrade was reverted in https://github.com/scalacenter/scalafix/pull/1083

Reply:
There is no real reason for that. I was just updating some libs, removing some todo, and adding the real change on top of that. 
I will remove that.